react-toggle-switch/my-app/src/App.tsx

```diff
@@ -0,0 +1,25 @@
+import './App.css';
+import './ToggleSwitch.css';
+import { ToggleSwitch } from './ToggleSwitch';
+import { useState } from 'react';
+
+export default function App() {
+  const [isOn, setIsOn] = useState(false);
+
+  const handleClick = () => setIsOn((prev) => !prev);
+
+  const className = `toggle-switch${isOn ? ' is-on' : ''}`;
```

Comment:
This would be best if it is inside the ToggleSwitch. You don't want to force the parent to think too much about things like which classname to use.